src/app/client/src/app/modules/certificate/components/certificate-details/certificate-details.component.ts

```diff
@@ -7,12 +13,69 @@ import { Component, OnInit } from '@angular/core';
 export class CertificateDetailsComponent implements OnInit {
   showSuccessModal: boolean;
   loader: boolean;
-  error: boolean;
   viewCertificate: boolean;
-  viewCertificateHtml: boolean;
+  error: boolean;
+  enableVerifyButton = false;
+  certificateCode: any;
```

Comment:
why is this any it should be a string right?

Reply:
done